doc/building.rst

```diff
@@ -215,21 +323,27 @@ part of the client source repository and not a separate module.
 
 To build the most up-to-date version of the client:
 
-1. Clone the latest versions of the client from Git_ as follows::
+1. Clone the latest versions of the client from Git_ as follows:
 
-     git clone git://github.com/nextcloud/client.git
-     cd client
-     git submodule init
-     git submodule update
+   .. code-block:: bash
 
-2. Create the build directory::
+      $ git clone git://github.com/nextcloud/client.git
+      $ cd client
+      $ git submodule init
+      $ git submodule update
```

Comment:
Could you remove these two lines that update the submodules?  They are not necessary.

Reply:
I'm not so sure, we do have a submodule! It's use should be repaired though, it's the one which would reenable having proper crash reports in the wild.

Also the two submodules steps could be compressed as: `git submodule update --init`.

Reply:
I just replaced the two lines with the consolidated one. FWIW that's actually a section I hadn't really touched, so what you're commenting was already there before I started my work. I had actually already removed that bit from the main macOS instructions, though I could easily re-add it.